installation/linux.md

```diff
@@ -184,22 +286,32 @@ The most important commands to control the openHAB service are given below.
   sudo systemctl enable openhab2.service
   ```
 
-#### Upgrade
+{% include collapsible/item-end.html %}
+{% include collapsible/end.html %}
+
+#### Upgrade/Downgrading
 
 To stay up to date with new releases, you should do regular upgrades.
 This is especially important if you are working with the latest snapshot as changes and fixes are incorporated constantly.
 
 Your personal configuration will be retained on upgrades.
 We still recommend a backup before each upgrade.
 
+{% include collapsible/start.html %}
+{% include collapsible/heading.html %}
+
+Apt Based Systems
+
+{% include collapsible/body.html %}
+
 Upgrading is as easy as:
 
 ```shell
 sudo apt-get update
 sudo apt-get upgrade
 ```
 
-#### Changing Versions
+*Changing Versions*
```

Comment:
I used to link to this headline. If we were to add an anchor here, would the collapsible open automatically?